Sources/GRPC/ClientConnection.swift

```diff
@@ -469,6 +469,24 @@ extension ClientConnection {
     @preconcurrency
     public var debugChannelInitializer: (@Sendable (Channel) -> EventLoopFuture<Void>)?
 
+    #if canImport(Network)
+    @available(macOS 10.14, iOS 12.0, watchOS 6.0, tvOS 12.0, *)
+    public var clientBootstrapNWParametersConfigurator: (
+      @Sendable (NIOTSConnectionBootstrap) -> Void
```

Comment:
We should be passing in `NWParameters` here (and elsewhere)

Reply:
Ah yeah this was me changing my mind a couple of times about whether we should allow the bootstrap to be configured or only the parameters - that's why the name of the property is a bit confused too.
I take from your previous comment that we should hide the concept of a bootstrap, so I'll stick to just customising the NWParameters then